### PR TITLE
aaded flag for aadhar verification for ui

### PR DESCRIPTION
**Pull request by AshishSaini25**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

## Description

added flag for aadhaar verification  for ui side
### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable

### CODE REVIEW

File: Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/CachedQueries/DriverInformation.hs [hunk 1]
```diff
@@ -46,6 +46,11 @@ updateEnabledState driverId isEnabled = do
   clearDriverInfoCache driverId
   Esq.runTransaction $ Queries.updateEnabledState driverId isEnabled
 
+updateAadhaarVerifiedState :: (CacheFlow m r, Esq.EsqDBFlow m r) => Id Person.Driver -> Bool -> m ()
+updateAadhaarVerifiedState driverId isVerified = do
+  clearDriverInfoCache driverId
+  Esq.runTransaction $ Queries.updateAadhaarVerifiedState driverId isVerified
```
Comment (piyushKumar-1) [inline Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/CachedQueries/DriverInformation.hs:52]: we can use runNoTransaction here.

File: Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Driver.hs [hunk 2]
```diff
@@ -178,6 +179,7 @@ data DriverEntityRes = DriverEntityRes
     enabled :: Bool,
     blocked :: Bool,
     verified :: Bool,
+    aadhaarVerified :: Bool,
     registeredAt :: UTCTime,
```
Comment (piyushKumar-1) [inline Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Driver.hs:183]: we were planning to add `aadhaarVerificationRequired` field in profile API response as well, can we add it ?

Comment (AshishSaini25) [reply Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Driver.hs:183]: yes
I have added in merchant

File: Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/CachedQueries/DriverInformation.hs (hunk 1 above)
Comment (AshishSaini25) [reply Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/CachedQueries/DriverInformation.hs:52]: ok

**Review by piyushKumar-1 — APPROVED** [review]

**Pull request merged**